clients/python/.gitignore

```diff
@@ -4,6 +4,7 @@
 /test
```

Comment:
This file contains `/airflow_client` , `/docs` and other things that we do need and are in-fact the source code :) so we can't use this file in https://github.com/apache/airflow-client-python. We could remove some of them manually but not worth it.

This file works as intended in the `apache/airflow` repo since we don't intend to commit python client files here